contents/blog/customer-support-at-posthog.md

```diff
@@ -0,0 +1,45 @@
+---
+date: 2021-06-15
+title: How We Do Customer Support At Our Open-Source DevTool Company
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+author: mo-shehu
+---
+<iframe width="560" height="315" src="https://www.youtube.com/embed/6giGcTSdkAc" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+## Thinner docs, better products
+
+The highest priority for any company is shortening the feedback loop between your customers and your product team. The fastest way to achieve this is by having your product builders interface directly with your customers (like we do at PostHog). This gives you an idea of the types of problems that keep cropping up and lets you decide how much help content you need to create - but most importantly, it tells you how to improve and simplify the product experience to reduce the need for thicker help docs and extensive tutorials.
+
+## A little background
+
+Our very first support staff members were the founders, James and Tim, due to the limited resources and staff we had at the time. Having all hands on deck made a big difference, as you can’t afford to designate customer support to just one person in the early days. 
+
+As of writing, customer support at PostHog happens in [“rotas”](https://posthog.com/handbook/growth/customer-support) - pre-planned, two-week cycles where one dev team member drops all development work and handles incoming user queries. Called a ‘Support Hero,’ this team member monitors all our channels - from GitHub and Slack to social media where applicable - and either solves the issues that surface or ropes in other team members for assistance. That last point is important, as having a support hero doesn’t abdicate everyone else from responding to customers. At PostHog, we firmly believe in “stepping on toes”: if you spot a problem that falls under someone else’s purview, just solve it.
```

Comment:
An intro point would be good here to say 'We do not hire support people at PostHog. Instead, support is something that _all_ of our engineers do on a rota basis. Yes, Tim as CTO still does it too.'

Reply:
We have one week rotas now.

Reply:
@mariusandra Amended